web/lib/components/nav/Settings.jsx

```diff
@@ -16,24 +18,44 @@ export function Settings() {
     voiceSettings.get() ?? voices?.[0]?.voiceURI,
   )
 
+  const {
+    settings: { showSpeechForExamples },
+    toggleShowExampleSpeech,
+  } = useContext(SettingsContext)
+
   const doSetVoice = (e) => {
     voiceSettings.set(e.target.value)
     setSelected(e.target.value)
   }
 
+  const doToggleExampleSpeech = (value) => toggleShowExampleSpeech(value)
```

Comment:
In this case `doToggleExampleSpeech` does nothing but calling `toggleShowExampleSpeech`, so they are kinda the same `doToggleExampleSpeech = toggleShowExampleSpeech`. For `onChange={doToggleExampleSpeech}` in ToggleCheckbox below we can do `onChange={toggleShowExampleSpeech}`